api/src/pcapi/core/offerers/models.py

```diff
@@ -1262,3 +1262,7 @@ class OffererAddress(PcObject, Base, Model):
     offerer: sa.orm.Mapped["Offerer"] = sa.orm.relationship("Offerer", foreign_keys=[offererId])
 
     __table_args__ = (sa.Index("ix_unique_offerer_address_per_label", "offererId", "addressId", "label", unique=True),)
+
+    @property
```

Comment:
Une `property` qui fait une query ne devrait pas être définie comme `hybrid_property` et avoir aussi sa définition en "python" ?

Reply:
Quand j'y repense maintenant on pourrait la définir comme une hybrid_property pour les cas où on voudrait filtrer sur cette condition. 
Je ne comprend pas bien le terme définition en "python". Est ce qu'il s'agit de la définition de l'expression ?

Reply:
Par exemple [ici](https://github.com/pass-culture/pass-culture-main/blob/7d4fca8c790aad4f542b0c6e10451821e722bc4b/api/src/pcapi/core/offers/models.py#L560). La méthode `canExpire` est définie une fois en utilisant l'opérateur built-in de python, et une deuxième fois avec l'opérateur sqlalchemy.